### ISSUE TITLE
testem started to break on travis because of missing launchers

### ISSUE THREAD

**Issue by MoOx** [airportyh/testem#589]
Previously, testem was only running on available browser that where specified in the config. It was really convenient because you can specify a large range of browsers to run your tests in several env with the same config file.
It seems that a recent change (that hasn't been released respecting semver (=> has a patch ?))  as introduced a breaking change in the public interface, so now some builds are simply not working anymore https://travis-ci.org/putaindecode/putaindecode.fr/builds/69086463

```
not ok 1 Error
    ---
        message: >
            Launcher safari not found. Not installed?
    ...
1..1
# tests 1
# pass  0
# fail  1
not ok 2 Error
    ---
        message: >
            Launcher chrome not found. Not installed?
    ...
1..2
# tests 2
# pass  0
# fail  2
not ok 3 Error
    ---
        message: >
            Launcher internet explorer not found. Not installed?
    ...
1..3
# tests 3
# pass  0
# fail  3
```

Is there a way to get this behavior back (in order to avoid having to maintain 2 config depending on the available environment)

**Comment by johanneswuerbach** [comment]
Sorry, I'll revert that asap. I had a misunderstanding with @airportyh and that should have been a BC bump.

**Comment by MoOx** [comment]
Thanks. Good to heard :)

**Comment by johanneswuerbach** [comment]
Revert, 0.8.5 should use the old behaviour. This change was made to prevent false-positives when no launcher was installed (testem exited with return code 0).

Just to clarify, you need the same `ci` setting to run in multiple environments?

**Comment by MoOx** [comment]
Currently I just tests all available browser (from my list), it means:
- locally I run all browsers, ff, chrome, safari,...
- on travis there is just ff & phantomjs, so...

Current behavior is nice, maybe we can change this by using a new flag or something ?

**Comment by johanneswuerbach** [comment]
Yep, makes sense to me.

**Comment by johanneswuerbach** [comment]
Included in `v0.9.0-1` as `ignore_missing_launchers` :-)
